editor/src/components/canvas/commands/push-intended-bounds-and-update-groups-command.ts

```diff
@@ -134,24 +137,38 @@ function getUpdateResizedGroupChildrenCommands(
       frameAndTarget.target,
     )
     if (targetIsGroup) {
-      const originalSize: Size = sizeFromRectangle(
-        MetadataUtils.getLocalFrameFromSpecialSizeMeasurements(
-          frameAndTarget.target,
-          editor.jsxMetadata,
-        ),
+      const children = MetadataUtils.getChildrenPathsOrdered(
+        editor.jsxMetadata,
+        editor.elementPathTree,
+        frameAndTarget.target,
       )
+
+      const originalSize: Size =
+        command.isMetadataStale === 'metadata-is-stale'
+          ? sizeFromRectangle(
```

Comment:
Minor: Could probably make this a little less hectic by extracting out into the conditional what we pass to `sizeFromRectangle`.